### ISSUE TITLE
Bug: Accessibility: 508 Keyboard Access Issues

### ISSUE THREAD

**Issue by binwang89** [18F/dol-whd-14c#789]
Bug description : this bug issue related to #627  #535 

1.	An interactive element or function cannot be accessed or activated by keyboard.
1.1 Example: https://dol-whd-section14c-qa.aisdemos.com/#!/dashboard - "clear all responses" function brings up a popup screen, it is not available for keyboard users.
1.2 Work Sites & Employees: Employee data (in the table) can not be sorted by keyboard.
Worked in Google Chrome, however, it is still not working in IE. 
2.	At any time, there is no visual indication of the current focus (loss of focus).
Example: Work Sites & Employees: The focus is lost after last "Delete" button and before "Back" button in the table.
Worked in Google Chrome, however, it is still not working in IE
3.	The visual focus does not move to revealed content and no description of the content change is provided .
3.1 Example: https://dol-whd-section14c-qa.aisdemos.com/#!/login
Create Account - When the user enters weak password, the screen provides instructions, however the focus does not fall on it, thus the screen reader does not read it out.
Not working in chrome/IE

4. The tabbing is not working for data table in this page in IE. #535

**Comment by NathayaMDOL** [comment]
@binwang89 ,
This story issues have been fixed except 1.2 which is in new story #793.
Branch: "whd-aqa-nmankong-789-508-a11y-keyboard-access"

Thanks,
Nathaya

**Comment by binwang89** [comment]
@NathayaMDOL Okay.  I will move it to next pipeline.

**Comment by binwang89** [comment]
@NathayaMDOL  I retested the data table area, it seems item 2, 4 are still not working.

**Comment by binwang89** [comment]
@NathayaMDOL @EStriegel @mmurthydol  re-tested item 2, 4 with different laptop ( same version IE browser), the issues have been fixed. The only issue left is 1.2  Work Sites & Employees: Employee data (in the table) can not be sorted by keyboard. Worked in Google Chrome, however, it is still not working in IE.  This issue also apply to any Data table (admin/User).  I will document it, and move this user story to  
closed.
